DOCS/interface-changes/tls-options.txt

```diff
@@ -0,0 +1,3 @@
+`--tls-ca-file` is now expanded as path
+`--tls-cert-file` is now expanded as path
+`--tls-key-file` is now expanded as path
```

Comment:
Sorry grammar again but this sounds weird to me. How about `the value of --tls-ca-file is now expanded as a path`?

Reply:
done